src/components/CohortBuilder/Categories/index.js

```diff
@@ -21,28 +21,107 @@ const Container = styled(Row)`
+    'phenotypes.ancestral_hpo_ids',
+    'diagnoses.diagnosis',
+    'diagnoses.diagnosis_category',
+    'family.family_compositions.composition',
+    'family.family_compositions.shared_phenotype_ids',
+    'outcome.is_disease_related',
+    'phenotype.hpo_phenotype_observed_text',
+    'diagnoses.source_text_tumor_location',
+    'outcome.vital_status',
+  ],
+  biospecimen: [
+    'biospecimen.age_at_event_days',
+    'biospecimen.analyte_type',
+    'biospecimen.ncit_id_anatomical_site',
+    'biospecimen.composition',
+    'biospecimen.method_of_sample_procurement',
+    'biospecimen.ncit_id_tissue_type',
+    'biospecimen.source_text_tumor_descriptor',
+  ],
+  availableData: [
+    'available_data_types',
+    'files.experiment_strategy',
+    'family.family_compositions.available_data_types',
+  ],
+};
+
+const Categories = ({ theme, sqon, onSqonUpdate }) => (
   <Container>
     <Search />
-    <Category title="Quick Filters" color={theme.filterPurple}>
+    <Category
+      title="Quick Filters"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.quickSearch}
+      color={theme.filterPurple}
+    >
       <QuickFilterIcon fill={theme.filterPurple} />
     </Category>
-    <Category title="Study" color={theme.studyRed}>
+    <Category
+      title="Study"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.study}
+      color={theme.studyRed}
+    >
       <StudyIcon fill={theme.studyRed} />
     </Category>
-    <Category title="Demographic" color={theme.demographicPurple}>
+    <Category
+      title="Demographic"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.demographic}
+      color={theme.demographicPurple}
+    >
       <DemographicIcon fill={theme.demographicPurple} />
     </Category>
-    <Category title="Clinical" color={theme.clinicalBlue}>
+    <Category
+      title="Clinical"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.clinical}
+      color={theme.clinicalBlue}
+    >
       <ClinicalIcon fill={theme.clinicalBlue} />
     </Category>
-    <Category title="Biospecimens" color={theme.biospecimenOrange}>
+    <Category
+      title="Biospecimens"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.biospecimen}
+      color={theme.biospecimenOrange}
+    >
       <BiospecimenIcon fill={theme.biospecimenOrange} />
     </Category>
-    <Category title="Available Data" color={theme.dataBlue}>
+    <Category
+      title="Available Data"
+      sqon={sqon}
+      onSqonUpdate={onSqonUpdate}
+      fields={CATEGORY_FIELDS.availableData}
+      color={theme.dataBlue}
+    >
       <FileIcon width={11} height={14} fill={theme.dataBlue} />
     </Category>
-    <Category title="Upload IDs" color={theme.uploadYellow}>
+
+    {/* the below is not actually a Category */}
+    <Category title="Upload IDs" color={theme.uploadYellow} fields={[]}>
```

Comment:
@ciaranschutte this one might be of interest for you later ;) it'll have a different functionality from the rest of the `Category`s :)